### DISCUSSION TITLE
Maybe Can add SVT-HEVC Encoder and SVT-VP9 Encoder

### DISCUSSION THREAD

**Discussion by KelvinSoh8** [staxrip/staxrip#831] [Ideas]
Compared With X265 and Google libvpx Encoder,The SVT-VP9 Encoder and SVT-HEVC Encoder Scalable Video Technology Encoder Intel optimized for CPU Processors, Which Get Greatly Improve  Encoder Speed Software Encoding.

**Comment by jessielw** [comment]
Last I checked it supported these encoders
